funnel/templates/schedule.html.jinja2

```diff
@@ -165,24 +234,24 @@
     var scheduleConfig = {
       fromDate: {{ from_date|tojson }},
       toDate: {{ to_date|tojson }},
-      scheduled: {{ sessions|tojson }},
+      timeZone: "{{ timezone }}",
+      sessions: {{ sessions|tojson }},
+      schedule: {{ schedule|tojson }},
       venues: {{ venues|tojson }},
       slotInterval: 5,
       divElem: "#project-schedule-table",
       scriptTemplate: '#scheduletemplate',
+      parentContainer: '#schedule',
       active_session: '',
       projectTitle: "{{ project.title }}",
```

Comment:
`{{ project.title|e|tojson }}` should work.

Reply:
Actually, `"{{ project.title }}"` is also fine. Since it will confuse someone else looking at this later, maybe it needs to be explained: `"{{ project.title|e }}", {#- HTML escaped for use in HTML tags via JS #}`